core/src/main/java/org/apache/carbondata/core/indexstore/blockletindex/BlockletDataMapFactory.java

```diff
@@ -364,6 +365,13 @@ public DataMapMeta getMeta() {
     return null;
   }
 
+  @Override
+  public boolean isSupport(Expression expression) {
+    // this method is used for choosing cg/fg datamap
+    // for default datamap always return false
+    return false;
```

Comment:
I get your point. I will change it to return true for better code reading. Also will add checking meta info( meta of default datamap is NULLnow) to avoid null point exception